packages/integration-karma/test/api/sanitizeAttribute/x/xlink/xlink.html

```diff
@@ -1,5 +1,5 @@
 <template>
     <svg>
-        <use xlink:href="/foo"></use>
+        <use xlink:href={xLink}></use>
```

Comment:
this is because of what [the test ](https://github.com/salesforce/lwc/blob/master/packages/integration-karma/test/api/sanitizeAttribute/index.spec.js)does. the test reset `LWC.sanitizeAttribute`, but because the hoisted content is evaluated just once, then it never executes `sanitizeAttribute`.